### ISSUE TITLE
problem generating PDF with make

### ISSUE THREAD

**Issue by baro77** [QISKit/ibmqx-user-guides#10]
doing:
make latex (in main directory)
then:
make (in <main directory>/_build/latex)

generate the PDF but with errors: one regarding unicode, many about nesting, many "Runaway argument?" 

Could you fix that? The generated PDF is pretty cool, so a look forward to have it generated in the correct way

Btw, downloaded and unzipped Master, TexMaC 2017 on macOS Sierra 10.12.6
thanks!

**Comment by jaygambetta** [comment]
This should be solved now. @francabrera can you confirm

**Comment by diego-plan9** [comment]
Yes, when compiling the `.tex` files there are commands that call `\sphinxincludegraphics` with filenames that have `%` in them - which is problematic, as it a latex reserved character. I'm exploring different approaches to solve it, currently looking at how to redefine the character code - ```\catcode`\%=12``` seems to do the trick, but I couldn't find yet how to apply it to just the blocks with the images instead of the full document.
